packages/drivers/routerlicious-driver/package.json

```diff
@@ -92,9 +92,9 @@
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../../.prettierignore",
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm test",
-		"test:mocha": "npm run test:mocha:esm && echo skipping cjs to avoid overhead - npm run test:mocha:cjs",
+		"test:mocha": "npm run test:mocha:esm && npm run test:mocha:cjs",
 		"test:mocha:cjs": "mocha --recursive \"dist/test/**/*.spec.*js\" --exit",
-		"test:mocha:esm": "mocha --recursive \"lib/test/**/*.spec.*js\" --exit",
+		"test:mocha:esm": "mocha --ignore \"lib/test/commonjs/*.spec.*js\" --recursive \"lib/test/**/*.spec.*js\" --exit",
```

Comment:
Shouldn't need this ignore since the tsconfig says not to build it. Be sure to clean a package anytime you remove or rename a file. The build system mostly makes sure output files exist and won't remove extra files.

Reply:
Wouldn't this ignore presist if tsconfig.cjs.json excludes are removed?

Reply:
It would. Would that be useful?

Reply:
This excludes cjs test files while esm testing.

Reply:
I don't think we want to bother building something that we don't use. The build exclusion for ESM seems like the right choice. Then no command line ignore.
I am also going to suggest that the files under commonjs be .cts. Those files are explicitly only CJS. They should never be otherwise. Part of the reason they should never be otherwise is that we cannot `require()` an ES Module.
Try this experiment:
1. remove the mocha ignore
2. change extension to .cts
3. clean (because you must after any rename)
4. build and test ESM

Reply:
after trying the above steps, build fails: `packages/drivers/routerlicious-driver/src/test/commonjs/mapWithExpiration.spec.cts:9:35 - error TS1479: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("../../mapWithExpiration.js")' call instead.`